lib/npg_qc/autoqc/results/rna_seqc.pm

```diff
@@ -1,17 +1,46 @@
-#########
-# Author:        Ruben Bautista
-# Created:       2015-08-13
-#
-
 package npg_qc::autoqc::results::rna_seqc;
 
 use Moose;
 use namespace::autoclean;
+use Readonly;
 
-extends qw(npg_qc::autoqc::results::result);
+extends 'npg_qc::autoqc::results::result';
 
 our $VERSION = '0';
-## no critic (Documentation::RequirePodAtEnd)
+
+Readonly::Array my @ATTRIBUTES => qw/ rrna
+                                      rrna_rate
+                                      exonic_rate
+                                      expression_profiling_efficiency
+                                      genes_detected
+                                      end_1_sense
+                                      end_1_antisense
+                                      end_2_sense
+                                      end_2_antisense
+                                      end_1_pct_sense
+                                      end_2_pct_sense
+                                      mean_per_base_cov
+                                      mean_cv
+                                      end_5_norm
+                                      end_3_norm
+                                    /;
+
+has [ @ATTRIBUTES ] => (
+    is         => 'rw',
+    isa        => 'Maybe[Num]',
```

Comment:
Would like to chat this particular case over. I think important here are
- reflecting what data we are capturing and 
- consistency serializing to the DB.